src/main/resources/assets/opencomputers/recipes/default.recipes

```diff
@@ -69,7 +69,7 @@ microcontrollerCase2 {
 }
 terminal {
   input: [[nuggetIron, "oc:solarGeneratorUpgrade", nuggetIron]
-          ["oc:circuitChip3", "oc:screen2", "oc:wlanCard"]
+          ["oc:circuitChip3", "oc:screen2", "oc:wlanCard2"]
           [nuggetIron, "oc:keyboard", nuggetIron]]
```

Comment:
I understand that the remote terminal doesn't need a wired message device but unless we are changing the available component in code of the crafted terminal to have a tier1 card - this crafting recipe could give the wrong impression. Also, do we also want to reduce the range of the remote terminal? Perhaps not, perhaps it is best to leave this as a tier 2 wireless card.